File: Desarrollos AddOn SAP B1/Samples/COM UI/CSharp/02.CatchingEvents/SubMain.cs
//   SAP MANAGE UI API 2007 SDK Sample
// ****************************************************************************
// 
//   File:      SubMain.cs
// 
//   Copyright (c) SAP MANAGE
// 
//  THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF
//  ANY KIND, EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO
//  THE IMPLIED WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A
//  PARTICULAR PURPOSE.
// 
// ****************************************************************************

using System;
using System.Windows.Forms;

class SubMain { 
    
    static void Main() { 
        
        //  Creating an object
        CatchingEvents oCatchingEvents = null; 
        
        oCatchingEvents = new CatchingEvents(); 
        
        //  Starting the Application
        System.Windows.Forms.Application.Run(); 
    } 
    
} 

File: Desarrollos AddOn SAP B1/Samples/COM UI DI/VB.NET/AddColumn/SubMain.vb
'//  SAP MANAGE UI,DI API 2006 SDK Sample
'//****************************************************************************
'//
'//  File:      SubMain.bas
'//
'//  Copyright (c) SAP MANAGE
'//
'// THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF
'// ANY KIND, EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO
'// THE IMPLIED WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A
'// PARTICULAR PURPOSE.
'//
'//****************************************************************************
Option Strict Off
Option Explicit On
Module SubMain

	Public Sub Main()
		
        Dim oAddColumn As AddColumn
		
        oAddColumn = New AddColumn

        Application.Run()
	End Sub
End Module